assignments/arrays.js

```diff
@@ -63,18 +63,30 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const car33 = inventory[32];
+console.log(car33);
 
```

Comment:
Your console log can be more detailed like how the example suggests.
Check out the link below on how to use `template literals`:
https://developers.google.com/web/updates/2015/01/ES6-Template-Strings